src/base/projects/View.svelte

```diff
@@ -43,6 +43,9 @@
     const seedInstance = profile?.seed ?? result?.seed;
     const cfg = seedInstance ? config.withSeed(seedInstance) : config;
     const info = await proj.getInfo(urn, cfg);
+
+    // Overwrite URN with a Radicle ID in case that the user has passed a project name
+    urn = isRadicleId(urn) ? urn : info.meta.urn;
```

Comment:
Can we split this so that the input is `export nameOrUrn: string`, then set `urn` from that?